packages/api/schema/stryker-core.json

```diff
@@ -168,19 +168,8 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "fileHeaders": {
-          "type": "object",
-          "title": "SandboxFileHeaders",
-          "description": "Configure additional headers to be added to files inside your sandbox. These headers will be added after Stryker has instrumented your code with mutants, but before a test runner or build command is executed. This is used to ignore typescript compile errors and eslint warnings that might have been added in the process of instrumenting your code with mutants. The default setting should work for most use cases.",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "default": {
-            "**/*+(.js|.ts|.cjs|.mjs)?(x)": "/* eslint-disable */\n// @ts-nocheck\n"
-          }
-        },
-        "stripComments": {
-          "description": "Configure files to be stripped of comments (either single line with `//` or multi line with `/**/`. These comments will be stripped after Stryker has instrumented your code with mutants, but before a test runner or build command is executed. This is used to remove any lingering `// @ts-check` or `// @ts-expect-error` comments that interfere with typescript compilation. The default setting allows comments to be stripped from all JavaScript and friend files in your sandbox, you can specify a different glob expression or set it to `false` to completely disable this behavior.",
+        "disableTypeChecking": {
```

Comment:
Does it still make sense to keep the `sandbox` options? Or should we place this on the root level of `StrykerOptions`?

We might want to add more preprocessor options in the future.

 @brodybits do you have an opinion?

Reply:
I'm leaning towards moving it to the toplevel 🤔

Reply:
I have very mixed feelings. I think keeping the option as part of `sandbox` would improve the abstraction. OTOH, it may add a little extra complexity.

Reply:
The thing is, "sandbox" is a term pinned by Stryker. It doesn't really mean anything for the average user. I've discussed introducing a `sandbox.mode` for the #2163 , but maybe that should just be called `--in-place` (boolean). 

Not adding the `sandbox` term in the configuration might make it easier to understand.

Reply:
I've removed it. Think it is the best option.